README.md

```diff
@@ -149,13 +149,13 @@ The function is called asynchronously, and it returns a `Result` object with a `
 
 #### Restore Purchases
 
-The `restorePurchases()` function helps to recover your app user's previous purchases without making them pay again. Sometimes, your app user may want to restore their previous purchases after switching to a new device or reinstalling your app. You can use the `restorePurchases()` function to allow your app user to easily restore their previous purchases.
+The `restorePurchases()` function helps to recover your app user's previous purchases without making them pay again. Sometimes, your app user may want to restore their previous purchases after switching to a new device or reinstalling your app. You can use the `restorePurchases()` function to allow your app user to easily restore their previous purchases by providing the `customer` object as a parameter.
 
 To retrieve **inactive** purchases along with the **active** purchases for your app user, you can call the `restorePurchases()` function with the `includeInactivePurchases` parameter set to true. If you only want to restore active subscriptions, set the parameter to false. Here is an example of how to use the restorePurchases() function in your code with the `includeInactivePurchases` parameter set to true.
 
 ``` dart
 try {
-  final result = await Chargebee.restorePurchases(true);
+  final result = await Chargebee.restorePurchases(true, customer);
```

Comment:
Can we define the customer object here like `final customer = CBCustomer('id','','','');`

Reply:
defined customer object can you please cross check @cb-haripriyan 
once you confirm will raise pr to next branch as well